.github/workflows/docker-GA-images.yml

```diff
@@ -0,0 +1,140 @@
+name: Build and Publish docker images for Hive GA
+
+on:
+  create:
+  workflow_dispatch:
+    inputs:
+      hiveVersion:
+        description: 'Hive version'
+        required: true
+        default: '4.0.0-beta-1'
```

Comment:
@simhadri-g Just thinking out loud. is there a way to not hard code the versions here? same for the hadoop and tez versions below. If there is a way to pull this from pom.xml or some other file (that gets generated as part of the build, then the docker image will be built for the latest bits.

If there isnt an easy way to do the above, would it make sense to version this file? for example docker-4.0.0-beta1-images.yml or something like that instead of GA in the name?